main/src/main/res/layout/search_activity.xml

```diff
@@ -180,8 +179,7 @@
             android:hint="@string/search_tb_hint" android:labelFor="@id/trackable">
             <AutoCompleteTextView
                 android:id="@+id/trackable"
-                style="@style/textinput_embedded_singleline_go"
-                android:inputType="textVisiblePassword" />
+                style="@style/textinput_embedded_singleline_go" />
```

Comment:
I read the linked issue and played around a bit with Android's textInput settings. In the end I came up with the following:

```
<AutoCompleteTextView
    android:id="@+id/trackable"
    style="@style/textinput_embedded_singleline_go"
    android:digits="ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
    android:inputType="textCapCharacters" />
```

This will show an alphanumeric keypad, allowing direct input of letters (automatically upper-casing them) and number input on long-press.

GC does not use all letters, but as I'm not sure which letters are allowed for the other connectors, I left in the full set A-Z.

Reply:
That restores the state before https://github.com/cgeo/cgeo/issues/7148 though? The point of textVisiblePassword is to force the number row, not just by long-press

Reply:
Found another solution. Number row is in again.